app/assets/stylesheets/semantic-ui/collections/_breadcrumb.scss

```diff
@@ -38,22 +38,23 @@
   opacity: 0.7;
   margin: 0em 0.21428571rem 0em;
   font-size: 0.92857143em;
-  color: rgba(0, 0, 0, 0.4);
+  color: $text-color;
   vertical-align: baseline;
 }
 
 /* Link */
 .ui.breadcrumb a {
-  color: #4183C4;
+  color: $white;
 }
 .ui.breadcrumb a:hover {
-  color: #1e70bf;
+  color: $white;
 }
 
 /* Icon Divider */
 .ui.breadcrumb .icon.divider {
   font-size: 0.85714286em;
   vertical-align: baseline;
+  color: $text-color;
```

Comment:
Is this really necessary? The text color will be inherited from L41 and they are the same anyway.